testplan/testing/multitest/driver/base.py

```diff
@@ -126,8 +126,22 @@ def uid(self):
         """Driver uid."""
         return self.cfg.name
 
+    def start(self):
```

Comment:
Start and stop logic already lives in Resource, from which Driver inherits. Can we move pre_start/pre_stop logic into Resource too?

If not, at least make a super() call here for the status-changing and starting() logic.